pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["setuptools>=42", "wheel"]   # PEP 508 specifications.
```

Comment:
I dislike that setuptools requires two files (``setup.cfg`` and ``toml``) when the ``toml`` can actually keep

For that reason, for a project of this size, i rather recommend using [flit](https://flit.readthedocs.io/en/latest/#usage). 

Just rename your current ``pyproject.toml`` (to avoid overridinng) and try
```
pip install flit
flit init
```

Reply:
I like merging all configs into one file. But flake8 for instance doesn't support toml files (yet), so I would need e.g. a .flake8 file: https://flake8.pycqa.org/en/latest/user/configuration.html

- this way I can put all options (except the build-system) into the setup.cfg. 
- do i even need build-system options?

Reply:
I guess its not hugely different, but I will give flint a try. Thanks